src/main/java/de/codingair/tradesystem/trade/Trade.java

```diff
@@ -218,6 +217,16 @@ public void onPickup(PlayerPickupItemEvent e) {
         }, TradeSystem.getInstance());
     }
 
+    private void dropItem(Player player, ItemStack itemStack) {
+        if (player == null || itemStack == null || itemStack.getType() == Material.AIR) return;
+        player.getWorld().dropItem(player.getLocation(), itemStack);
+
+        String msg = Lang.get("Item_Dropped", player)
+                .replace("%amount%", String.valueOf(itemStack.getAmount()))
+                .replace("%item%", itemStack.getType().name());
+        player.sendMessage(msg);
```

Comment:
This would result many many messages on a full trade with a full inventory. Might be better to remove that. What do you think?

Reply:
We might be able to provide a summary at the end, so different items would get different messages, but the same items will be in the same message, but unsure. I do think we should somehow notify the player an item was dropped on the ground, maybe we can even just display one message “items were dropped on the ground” instead of all items that were dropped.

Reply:
I like the last solution. Just one message for "items were dropped ...".

Reply:
Done! 👍

Reply:
Awesome!